.rubocop.yml

```diff
@@ -4,6 +4,10 @@ AllCops:
     - 'tmp/**/*'
     - 'vendor/**/*'
 
+# FIXME: Re-enable once private_attr_reader is no longer used
+Lint/UselessAccessModifier:
```

Comment:
But then we could easily fix this by moving those declarations up like [here](https://github.com/troessner/reek/blob/master/lib/reek/source/source_locator.rb#L32) and then remove this configuration setting, can't we?
If yes, I'd prefer to do this in this  PR since it's also only a couple of occurences.